script/upload.py

```diff
@@ -72,6 +72,8 @@ def main():
     # Press the publish button.
     publish_release(github, release['id'])
 
+    # TODO: run publish-to-npm script here
```

Comment:
@vanessayuenn do you think we should leave this as-is and publish manually for now? Or would you rather implement it before we ship?